docs/support/faq.md

```diff
@@ -136,97 +124,81 @@ Yes! See [boolean logic](../sdk/access-control/condition-types/boolean-logic) fo
 
-Yup! those are the default params we are launching with. Over time, we want to let users launch their own "subnets" with their own parameters.
+### 6. So you need 2/3 of the entire network to decrypt or sign the content? Not 2/3 of some fixed constant number of key fragments?
 
-### 8. As the number of nodes on the network grows, it gets more secure, but also slower to decrypt content?
+Yup! Those are the default params we are launching with. Over time, we want to let users launch their own "subnets" with their parameters.
 
-Yeah that's correct. we plan to tune the network to have the max number of nodes while still remaining within some performance bounds. like if we can have 100 nodes and it takes less than 2 seconds to unlock something, that would be acceptable. when the network grows beyond that size, we support the automatic creation of subnets, which are basically just parallel networks. and then when someone goes to store some content, automatically load balance between those subnets
+### 7. How is performance impacted as the number of nodes in the network increases?
 
-### 9. So long as that an attack on the network remains impractical then the system is pretty robust?
+The network is designed to handle a large number of nodes while maintaining acceptable performance. As the network grows, subnets can be automatically created to distribute the load and improve performance.
 
-**(cont'd) One operator runs enough nodes to gather sufficient fragments to decrypt stuff - is sufficiently difficult and costly to execute so as to be impractical. That still seems like the most likely attack vector if things aren't designed just right. **
+### 8. Is the system robust against attacks?
 
-Yeah you nailed it - there are lots of little tricks we can use there. like suppose there are 10 subnets each with their own private key, each with 100 nodes. if you wanted to break one of those private keys, you would need to run 66 of 100 of those nodes. but, when you join the network, the subnet you are assigned to is uniformly random. so now, you need to run 666 of 1000 nodes to amass those 66 shares for a single given subnet (maybe technically less due to the birthday problem). we could give out fake shares. we could make it difficult to discover which public key goes with which private key share, so as a node operator can't even tell which shares will unlock a given piece of data. It's a hard problem to solve but we have a lot of tools.
+The system is designed to mitigate attacks, including the scenario where an operator runs multiple nodes to gather sufficient fragments to decrypt data. Various techniques are employed, such as distributing private keys across subnets and making it difficult to determine which shares correspond to specific data. These measures make such attacks impractical and costly to execute.
```

Comment:
again, where security is concerned i think we should focus on mitigations that have already been implemented vs hypotheticals. In this case, let's just discuss the distributed nature of the network (multiple node operators), threshold consensus (need 2/3), and secure hardware (operators don't have access to key shares themselves)